walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/style.css

```diff
@@ -4,22 +4,33 @@
   width: 650px;
 }
 
-.image-thumbnails {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
+/* Add a black background color to the top navigation */
+.menuBarClass {
+  background-color: #FFFFFF;
+  overflow: hidden;
 }
 
-.image-thumbnails img {
-  margin: 10px;
-  max-height: 200px;
-  max-width: 200px;
+/* Style the links inside the navigation bar */
+.menuBarClass a {
+  /*float: left;*/
+  color: 	#000000;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+  display: flex;
+  align-items: center;
+  justify-content: center
+  /*margin-left: auto;
+  margin-right: auto;*/
 }
 
-.image-thumbnails img:hover {
-  border: thick solid black;
+/* Change the color of links on hover */
+.menuBarClass a:hover {
+   text-decoration: underline;
 }
 
-#random-image-container img {
-  margin-top: 20px;
+/* Hide the link that should open and close the topnav on small screens */
+.menuBarClass .icon {
+  display: none;
 }
```

Comment:
Right now you're hiding it all the time. Perhaps it's better to add the button when you're actually going to implement it rather than leaving this here to potentially become dead code.